src/migrations/20200707130924_password_reset_token_source.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class PasswordResetTokenSource extends AbstractMigration
+{
+    public function change()
+    {
+        $this->table('password_reset_tokens')
+            ->addColumn('source', 'string', ['null' => true, 'default' => null])
+            ->create();
```

Comment:
Can we use `update()` here?

- `create()` works too but it will quietly create new table when table `password_reset_tokens` doesn't exists _(and it will be missing other columns)_,
- `update()` adds new column; migration fails in case table doesn't exist _(preferred outcome)_.

Reply:
We are a bit more explicit in migrations so they fail when something is not in the state we expect it should be.

- We `create()` only when we actually want to "create table"
- We `update()` only when we actually want to "update existing table"
- We never call `change()` because it's ambigous :).

_(I'll make the change internally, I have the patch ready. No need to push anoter commit)._

Reply:
Fixed